db/migrate/20160605223824_distinguish_starters_from_relievers.rb

```diff
@@ -0,0 +1,17 @@
+class DistinguishStartersFromRelievers < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+      CREATE TYPE pitcher_type AS ENUM('SP', 'RP', 'SP/RP', 'RP/SP');
+    SQL
+ 
+    add_column :pitchers, :pitcher_type, :pitcher_type, null:false, after: :last_name, default: "RP"
```

Comment:
The `after` does not work with Postgres. There are clunky workarounds but a pretty column order is not worth the hassle at this point.